packages/mobile/src/components/user-badges/UserBadges.tsx

```diff
@@ -28,11 +29,16 @@ export const UserBadges = (props: UserBadgesProps) => {
     mint: 'BONK'
   })
 
+  const shouldShowArtistCoinBadge =
+    isArtistCoinEnabled &&
+    !(env.ENVIRONMENT === 'production' && userId === 51) &&
+    !(env.ENVIRONMENT === 'staging' && userId === 12372)
```

Comment:
maybe I'm misunderstanding. Do we want to hide audio badges in general or just for the artist coin owner (audius account)?

Reply:
oh good point..

Reply:
yea, think we can just filter the mint instead of doing this approach